lang/cs/BUILD.md

```diff
@@ -24,7 +24,13 @@ Prerequisites
 -------------
 
   * Windows OS.
-  * [Building REEF Java](../java/BUILD.md).
+  * Apache maven
```

Comment:
We should be more specific here and not endorse specific vendors. To build REEF, one needs the JDK, not just the JRE. Also, We need Maven 3.3 or newer.

How about the following:

```
Prerequisites
-------------

  * Windows OS. We have had success with Windows 8, 10, Server 2012 and Server 2016.
  * All the same prerequisites as for REEF Java ([Building REEF Java](../java/BUILD.md)), namely:
    * Java Development Kit 7 or 8. Make sure that the environment variable `JAVA_HOME` points to its installation folder. We have had success with both Oracle's JDK and builds of the OpenJDK.
    * [Apache Maven](https://maven.apache.org/download.cgi) 3.3 or newer. Make sure that the environment variable `M2_HOME` points to its installation.
    * [Protocol Buffers Compiler version 2.5](https://github.com/google/protobuf/releases/tag/v2.5.0).
      Make sure that `protoc.exe` is in a folder referenced by your `$PATH` environment variable.
  * [Visual Studio](http://www.visualstudio.com) 2015 (preferred) or 2013. Most REEF developers use the free Community Edition.
  * NuGet 2.8.6 or later. (Included in VS 2015)
  * xunit.runner.console.2.1.0 package (installing it might require manual restore of NuGet packages).
```